tmd/io/io.py

```diff
@@ -122,6 +122,9 @@ def load_population(neurons, tree_types=None, name=None):
 
     pop = Population.Population(name=name)
 
+    if any([i.endswith(".asc") for i in files]):
+        raise Exception('You have some .asc files which cannot be used with TMD, please only use .h5 or .swc.')
```

Comment:
Maybe this should be a warning instead of an error (in case there are both type of files in the some directory).